pxr/imaging/plugin/hdRpr/volume.cpp

```diff
@@ -1,6 +1,9 @@
 #include "volume.h"
-
+#pragma warning(push)
+#pragma warning(disable:4146)
 #include "openvdb/openvdb.h"
+#pragma warning(pop)
```

Comment:
I am in favor of removing this. So as it's MSVC only solution and actually we should not care about warnings coming from external headers. Maybe we could contribute to openvdb repository to fix this warning, ofc if it's possible

Reply:
Removed.